src/main/java/Task.java

```diff
@@ -0,0 +1,34 @@
+public class Task {
+    private String task;
+    private boolean done;
```

Comment:
A better way to name boolean variables would be isDone or hasDone!

Reply:
Oh yes!